.vib/apache/goss/goss.yaml

```diff
@@ -14,9 +14,10 @@ file:
     {{ end }}
     {{ end }}
 command:
+  {{- $uid := .Vars.containerSecurityContext.runAsUser }}
+  {{- $gid := .Vars.podSecurityContext.fsGroup }}
   check-user-info:
-    exec: id
+    # The UID and GID should always be either the one specified as vars (always a bigger number that the default)
+    # or the one randomly defined by openshift (larger values). Otherwise, the chart is still using the default value.
+    exec: if [ $(id -u) -lt {{ $uid }} -o $(id -G | awk '{print $2}') -lt {{ $gid }} ]; then exit 1; fi
```

Comment:
According to [shellcheck](https://www.shellcheck.net/wiki/SC2166), this syntax is marked as deprecated and should be avoided. Instead, favor `if [ a ] || [ b ];`.

Reply:
Applied, thanks!